src/model/input.ts

```diff
@@ -8,129 +8,195 @@ const core = require('@actions/core');
  * Note that input is always passed as a string, even booleans.
  */
 class Input {
+  public static githubEnabled = true;
+  public static cliOptions;
+  static get cloudRunnerTests(): boolean {
+    return Input.getInput(`CloudRunnerTests`) || false;
+  }
+  private static getInput(query) {
+    return Input.githubEnabled
+      ? core.getInput(query)
+      : Input.cliOptions !== undefined && Input.cliOptions[query] !== undefined
+      ? Input.cliOptions[query]
+      : process.env[query] !== undefined
+      ? process.env[query]
+      : false;
+  }
+  static get region(): string {
+    return Input.getInput('region') || 'eu-west-2';
+  }
+  static get githubRepo(): string {
+    return Input.getInput('GITHUB_REPOSITORY') || 'game-ci/unity-builder';
+    // TODO system get repo remote?
```

Comment:
Leftover or still todo?

Reply:
Will implement, was still todo

Reply:
This comment refers to the behavior of getting the github repository remote url from git cli. Useful for some cli scenarios to avoid having to require the input